src/quantuminspire/api.py

```diff
@@ -501,6 +501,7 @@ def _create_job(self, name: str, asset: Dict[str, Any], number_of_shots: int,
             The properties describing the new job.
             See :meth:`~.get_job` for a description of the job properties.
         """
+        full_state_projection = full_state_projection and not backend_type.get("is_hardware_backend", False)
```

Comment:
Shouldn't the fallback value be `True`. If we don't know whether it is a hardware_backend or not (for whatever value), safest would be to default to see the algorithm as non-deterministic?

Reply:
That's exactly why the default is False, non-deterministic (full_state_projection = False). See also knowledge base, optimizations of simulation: "Full state projection is the keyword used internally in Quantum Inspire to do optimization using a single determination of the full final state. By default, this argument is set to False"

Reply:
!! I see, you are right, I will change to True, I overlooked the not